commitizen/cli.py

```diff
@@ -597,8 +597,33 @@ def parse_no_raise(comma_separated_no_raise: str) -> list[int]:
     return no_raise_codes
 
 
+if TYPE_CHECKING:
+
+    class Args(argparse.Namespace):
+        config: str | None = None
+        debug: bool = False
+        name: str | None = None
+        no_raise: str | None = None
```

Comment:
We could probably add a comment mentioning that it will later be parsed as `list[int]`. A bit confusing at the first glance